tests/integration_tests/core_tests.py

```diff
@@ -615,7 +615,7 @@ def test_cache_logging(self):
         app.config["STORE_CACHE_KEYS_IN_METADATA_DB"] = store_cache_keys
 
     def test_redirect_invalid(self):
-        model_url = models.Url(url="hhttp://invalid.com")
+        model_url = models.Url(url="not-http://invalid.com")
```

Comment:
```suggestion
        model_url = models.Url(url="http://invalid.com")
```

Reply:
I think it's actually trying to test w/ an invalid protocol.

If it isn't, then it needs way more text explaining what it's trying to do...

Reply:
Haha... I didn't even look at it that closely... then disregard my suggestion - your change is fine! I thought it was a really strange correction.

Reply:
Well, there were two, and I guessed the same way for both. It seems it was a split decision...